elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -157,7 +157,8 @@ public FilterExpression parseGlobalExpression(String path, MultivaluedMap<String
 
                 try {
                     Node ast = parser.parse(expressionText);
-                    RSQL2ExpressionFilterVisitor visitor = new RSQL2ExpressionFilterVisitor(false);
+                    // this is not guarded against to-many joins
+                    RSQL2ExpressionFilterVisitor visitor = new RSQL2ExpressionFilterVisitor(true);
```

Comment:
This behavior will allow nested filters on 'to many' relationships as well.  This isn't the same behavior as the default filter dialect.